sklearn/feature_extraction/image.py

```diff
@@ -322,11 +322,10 @@ def extract_patches_2d(image, patch_size, max_patches=None, random_state=None):
         of patches.
 
     random_state : int, RandomState instance or None, optional (default=None)
-        Pseudo number generator state used for random sampling to use if
-        `max_patches` is not None.  If int, random_state is the seed used by
-        the random number generator; If RandomState instance, random_state is
-        the random number generator; If None, the random number generator is
-        the RandomState instance used by `np.random`.
+        The random number generator is used for random sampling when
```

Comment:
We usually phrase this to refer to what the parameter does, i.e. "Determines the random number generator used for ..."

Reply:
Thanks ..edited where relevant